subprojects/antlr/src/main/java/org/gradle/api/plugins/antlr/AntlrPlugin.java

```diff
@@ -68,16 +68,20 @@ public void apply(final Project project) {
         // Wire the antlr configuration into all antlr tasks
         project.getTasks().withType(AntlrTask.class).configureEach(antlrTask -> antlrTask.getConventionMapping().map("antlrClasspath", () -> project.getConfigurations().getByName(ANTLR_CONFIGURATION_NAME)));
 
-        project.getExtensions().getByType(JavaPluginExtension.class).getSourceSets().all(
+        JavaPluginExtension javaPluginExtension = project.getExtensions().getByType(JavaPluginExtension.class);
+        javaPluginExtension.getSourceSets().all(
                 new Action<SourceSet>() {
                     @Override
                     public void execute(final SourceSet sourceSet) {
                         // for each source set we will:
                         // 1) Add a new 'antlr' virtual directory mapping
-                        final AntlrSourceVirtualDirectoryImpl antlrDirectoryDelegate
-                                = new AntlrSourceVirtualDirectoryImpl(((DefaultSourceSet) sourceSet).getDisplayName(), objectFactory);
+
+
+                        final org.gradle.api.plugins.antlr.internal.AntlrSourceVirtualDirectoryImpl antlrDirectoryDelegate
+                                = new org.gradle.api.plugins.antlr.internal.AntlrSourceVirtualDirectoryImpl(((DefaultSourceSet) sourceSet).getDisplayName(), objectFactory);
```

Comment:
Where did the import go?

Reply:
I think we could also remove `final` as it doesn't really add anything, but that's extra. :)

Reply:
I deprecated `AntlrSourceVirtualDirectoryImpl` in this PR and our build complains about its usage unless I:
- add @SuppressWarnings("deprecation") statement on the class (or method)
- remove the import statement